src/current/v24.1/upgrade-cockroach-version.md

```diff
@@ -12,6 +12,9 @@ docs_area: manage
 {% assign previous_latest_prod = site.data.releases | where_exp: "previous_latest_prod", "previous_latest_prod.major_version == previous_version" | where: "release_type", "Production" | sort: "release_date" | last %}
 {% assign actual_latest_prod = site.data.releases | where: "major_version", site.versions["stable"] | where: "release_type", "Production" | sort: "release_date" | last %}
 
+{% capture previous_version_numeric %}{{ previous_version | strip_first: 'v'}}{% endcapture %}
+{% capture major_version_numeric %}{% assign local_major_version = page.major_version %}{{local_major_version | strip_first: 'v' }}{% endcapture %}
```

Comment:
Maybe this will work? The var isn't displaying a value currently.

```suggestion
{% capture major_version_numeric %}{% assign local_major_version = page.version.version %}{{local_major_version | remove_first: 'v' }}{% endcapture %}
```